src/measure_theory/bochner_integration.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhouhang Zhou
+
+Bochner integral on real normed space
```

Comment:
You coud maybe extend a little bit the comments, giving some references and emphasizing that you only work with second-countable spaces (Bochner integrals can be defined for more general spaces, but being Bochner-integrable is then more restrictive than being measurable, and the Bochner-integrable functions take values in second-countable subspaces, so arguably the only interesting case is the second-countable one).